op-e2e/system/e2esys/setup.go

```diff
@@ -620,6 +620,13 @@ func (cfg SystemConfig) Start(t *testing.T, startOpts ...StartOption) (*System,
 		}
 	}
 
+	var l2BlobConfig *rollup.L2BlobConfig
+	if c, ok := cfg.Nodes["sequencer"]; ok && c.DACConfig != nil {
+		l2BlobConfig = &rollup.L2BlobConfig{
```

Comment:
`rollup.L2BlobConfig` should be configured by `DeployConfig.L2BlockTime`, not by `DACConfig`. In theory, user can set `rollup.L2BlobConfig` with or without `DACConfig`.